apps/framework-cli/src/cli/commands.rs

```diff
@@ -52,6 +52,13 @@ pub enum Commands {
         /// Build for arm64 architecture
         #[arg(long)]
         arm64: bool,
+        /// Expose Dockerfile to project root for customization
+        /// generates Dockerfile at ./Dockerfile instead of hidden .moose/packager/
+        #[arg(long, requires = "docker", conflicts_with = "no_expose_dockerfile")]
+        expose_dockerfile: bool,
+        /// Revert to hidden, managed Dockerfile (default behavior)
+        #[arg(long, requires = "docker", conflicts_with = "expose_dockerfile")]
+        no_expose_dockerfile: bool,
```

Comment:
I find these arguments to be a bit strange to use and since we are adding this to our public interface I am hesitant since any changes here would be breaking. 

I wonder if we let users configure it via moose.config.toml where they can enable and set the path of the dockerfile for their project and then provide a way to generate it via moose init? Essentially, overriding the config would move moose from using the file in .moose/packager to the one specified by the moose config for each moose app.